.gitignore

```diff
@@ -6,3 +6,6 @@ Gemfile.lock
 
 # SimpleCov
 coverage
+
+# Temp
+null.txt
```

Comment:
@buys-fran Weird. What's this?

Reply:
In the spec helper I am "redirecting" stdout to a null.txt file.
That way you can view the warning if you wish. But it does not muddy up the spec output.
This is because the gem itself prints to stdout - which makes it harder to see the spec output.

Reply:
maybe we can redirect the output to a file inside tmp? though I understand you removed that redirect part